### PR TITLE
Botonic scripts: Translate bash scripts to Typescript #SER-2405

### PR DESCRIPTION
**Pull request by alexgimenez02**
## Description
To mantain all scripts in a single language, the bash scripts are translated to Typescript scripts.
What is missing: 

- [x]  Update package.json to use ts scripts
- [x]  Translate scripts in CI folder
- [x]  Update pre-commit-config.yaml to use the ts scripts
- [x]  ~~Remove `.sh` scripts from the folders (once this PR is approved)~~ Move old scripts into a folder `old` for each respective original folder (`qa` or `ci`)

## Approach taken / Explain the design
Adapt the previous bash scripts to typescript script in `scripts` folder

## To document / Usage example
To run the scripts, use `npx ts-node <script-name>.ts <argument>`, or simply `ts-node <script-name>.ts <arguments>`.
Ideally, an npm script will be added to package.json to easen the task of running the scripts.

### CODE REVIEW

**Review by vanbasten17 — APPROVED** [review]
💯

File: scripts/ci/bump-version.ts [hunk 1]
```diff
@@ -30,16 +30,16 @@ try {
...
+const version = process.argv[3]
+const phase = process.argv[4]
+if (!!phase) {
   const PACKAGES_DIR: string = path.resolve(`${BIN_DIR}/../../packages`)
 
   const packageInPackages = fs.readdirSync(PACKAGES_DIR)
   const botonicPackages = packageInPackages.filter(function (file: string) {
     return file.startsWith('botonic')
   })
+  console.log(process.cwd())
```
Comment (asastre) [inline scripts/ci/bump-version.ts:42]: Remove `console.log`?

**Review by asastre — APPROVED** [review]

**Comment by vanbasten17** [comment]
@alexgimenez02 , I think it can be already merged ;-)

**Comment by alexgimenez02** [comment]
> @alexgimenez02 , I think it can be already merged ;-)

I will push some changes to fix the GH actions and merge!

**Pull request merged**